.github/workflows/deploy-to-obs-landing.yaml

```diff
@@ -0,0 +1,55 @@
+name: deploy-to-obs-landing
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build-obs-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup SSH Client
+        uses: webfactory/ssh-agent@v0.5.4
+        with:
+          ssh-private-key: ${{ secrets.DEPLOY_KEY_PRIVATE }}
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ssh-key: ${{ secrets.DEPLOY_KEY_PRIVATE }}
+      - name: Checkout obs-landing repo
+        uses: actions/checkout@v3
+        with:
+          repository: openSUSE/obs-landing
+          path: obs-landing
+          ssh-key: ${{ secrets.DEPLOY_KEY_PRIVATE }}
+      - name: build
+        uses: addnab/docker-run-action@v3
+        with:
+          image: openbuildservice/obs-docu:latest
+          options: -v ${{ github.workspace }}:/github/workspace/
+          run: |
+            pushd /github/workspace/
+            for doc in obs-admin-guide obs-user-guide; do
+              ROOTID=${doc%-guide}
```

Comment:
Sorry, but this isn't a good idea to set the `ROOTID` like this. This parameter is defined inside the DC file or not at all. It points to the ID of the root element. Maybe set it in your DC file.

No need to explicitly set it here, so I would recommend to remove it.

```suggestion
```

Reply:
@tomschr thanks for the feedback, I removed it, I didnt saw it is defined in the DC files :)